extensions/jsonrpc/src/main/java/jolie/net/JsonRpcProtocol.java

```diff
@@ -381,14 +395,14 @@ private CommMessage recv_createCommMessage( int messageSize, byte[] messageConte
 					"/", value.getFirstChild( "params" ), null );
 			} else if( value.hasChildren( "error" ) ) {
 				String operationName = jsonRpcOpMap.get( jsonRpcId );
-				return new CommMessage( Long.parseLong( jsonRpcId ), operationName, "/", null,
+				return new CommMessage( jsonRpcId.hashCode(), operationName, "/", null,
```

Comment:
What's the reason for this change?

Reply:
We should not silently ignore `NumberFormatExceptions`, but I have fixed this in the new version of the commit.